gno.land/pkg/gnoclient/util.go

```diff
@@ -20,3 +20,10 @@ func (msg MsgCall) validateMsgCall() error {
 	}
 	return nil
 }
+
+func (msg MsgRun) validateMsgRun() error {
+	if msg.Package == nil || len(msg.Package.Files) == 0 {
```

Comment:
Maybe use `msg.Package.Validate()`? (This should probably be done for MsgCall as well).

The check for `if package == nil || len(files) == 0` should be added at the top of Validate, BTW, this way we catch these also server-side.

Reply:
I'll keep this this way for now, and I'll open up another PR to add this to `Validate` within the VM.